composer.json

```diff
@@ -9,7 +9,7 @@
     },
     "require-dev": {
         "brain/monkey": "2.4.0",
-        "dealerdirect/phpcodesniffer-composer-installer": "0.5.0",
+        "dealerdirect/phpcodesniffer-composer-installer": "0.7.0",
```

Comment:
This is the main change that allows Composer 2.0

Reply:
Does your comment mean that version 0.5.0 of this package was failing with composer 2.0?

Reply:
Yes, I think `0.5.0` caused this message:

```sh
$ git checkout develop
$ composer self-update --2 # If needed, changes Composer to 2.*
$ composer install
Installing dependencies from lock file (including require-dev)
Verifying lock file contents can be installed on current platform.
Your lock file does not contain a compatible set of packages. Please run composer update.

  Problem 1
    - dealerdirect/phpcodesniffer-composer-installer is locked to version v0.5.0 and an update of this package was not requested.
    - dealerdirect/phpcodesniffer-composer-installer v0.5.0 requires composer-plugin-api ^1.0 -> found composer-plugin-api[2.0.0] but it does not match the constraint.
```